### PR TITLE
First draft of the Finance Section

### PR DESCRIPTION
**Pull request by aaryapatil**
The first draft of the finance section is ready. I am still new to the finances, so there is a possibility I have forgotten a grant/funding or have some incorrect numbers/info (Sorry about that!). Would appreciate feedback from the finance committee @eteq @kelle @jonathansick (and @hamogu).

TODO: Add a few sentences talking about the upcoming finance APE.

### CODE REVIEW

**Comment by aaryapatil** [comment]
Not sure why the build is failing. The make works locally.

**Comment by pllim** [comment]
Perhaps the failure is unrelated. I see other PRs failing too.

**Comment by pllim** [comment]
Please rebase to remove unrelated failure. Thanks!

Is this ready for review?

File: src/ms.tex [hunk 1]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
+was available for the project development, and NUMFOCUS covered the incurred
```
Comment (jdswinbank) [inline src/ms.tex:301]: NumFOCUS didn't cover all (or any?) incurred costs.

File: src/ms.tex [hunk 2]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+operational costs. However, transitioning to long-term financial stability meant
+having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},
```
Comment (jdswinbank) [inline src/ms.tex:303]: Remove citation.

File: src/ms.tex [hunk 3]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
```
Comment (eteq) [inline src/ms.tex:296]:
```suggestion
This committee oversee planning and allocation of finances on behalf
```
maybe it's just me, but "disbursement" seems a bit less community-centric, and this sounds better to me

File: src/ms.tex [hunk 4]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+of the project. The Astropy project accepts funds from institutions as well as
```
Comment (eteq) [inline src/ms.tex:297]:
```suggestion
of the project. The Astropy Project accepts funds from institutions as well as
```

File: src/ms.tex [hunk 5]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
 
 \secauthor{Aarya Patil}
 
-Summarize funding sources (Moore, NASA) and amounts and what this has been used
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
```
Comment (eteq) [inline src/ms.tex:294]:
```suggestion
Project, the Coordination Committee under the new governance charter (see \S
```

File: src/ms.tex [hunk 6]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+the fiscal sponsor of the project. Previously, no direct financial support
+was available for the project development, and NUMFOCUS covered the incurred
+operational costs. However, transitioning to long-term financial stability meant
+having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},
+and hence the Astropy project applied for and successfully acquired several grants.
+The two major ones are briefly summarized below.
 
+\begin{itemize}
+    \item Moore: The Gordon and Betty Moore Foundation awarded the Astropy
+    project a \$900,000 grant in 2019 for the proposal ``Sustaining and Growing
```
Comment (jdswinbank) [inline src/ms.tex:309]: \$900,000 (US).

File: src/ms.tex [hunk 7]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+    maintainers through mentorship efforts, (3) the formalization of a governance
+    structure (see \S \ref{sec:project-governance}), and (4) the improvement
+    of equity, diversity, and inclusion efforts in the project (see \S
+    \ref{sec:project-ide}). Some of the money supported travel for meetings,
+    conferences, and workshops. This generous grant helped reduce reliance on
+    volunteer-driven maintenance of the openly-developed package and paved
+    the way to acquire funding from federal agencies: a big milestone for the
+    success of the project.
+
+    \item NASA: A successful proposal to the NASA ROSES-2020 call, section E.7
```
Comment (jdswinbank) [inline src/ms.tex:323]: Footnote with reference to the call?

File: src/ms.tex [hunk 8]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
-for.
+% Summarize funding sources (Moore, NASA) and amounts and what this has been
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
```
Comment (jdswinbank) [inline src/ms.tex:298]: NUMFOCUS capitalization.

File: src/ms.tex [hunk 9]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
+was available for the project development, and NUMFOCUS covered the incurred
+operational costs. However, transitioning to long-term financial stability meant
+having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},
+and hence the Astropy project applied for and successfully acquired several grants.
```
Comment (eteq) [inline src/ms.tex:304]:
```suggestion
and hence the Astropy Project applied for and successfully acquired several grants.
```

Comment (jdswinbank) [inline src/ms.tex:304]: Is “several” == “four”? I feel like if we know there are four grants we can enumerate below, we should say that.

File: src/ms.tex [hunk 10]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+    Astropy ecosystem and educational materials (see \S \ref{sec:learn}),
+    and (4) support for the \astropy affiliated packages. These goals
+    support the sustenance of the project by encouraging further
+    development of the package ecosystem and community engagement.
+\end{itemize}
+
+In addition to the two grants described above, the project benefited from
+a Gemini Observatory contract under its Science User Support Department,
+which funded development work to support both the Astropy and DRAGONS
+projects. The project also received a total of CAD 31,023 from the Dunlap
```
Comment (jdswinbank) [inline src/ms.tex:338]: $31,023 (Canadian).

**Review by jdswinbank — APPROVED** [review]
Comments captured during Finance Ctte discussion.

File: src/ms.tex [hunk 11]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+% used for.
+
+With the goal of establishing a sustainable financial model for the Astropy
+project, the Coordination Committee under the new governance charter (see \S
+\ref{sec:project-governance}) established a standing Finance committee.
+This committee overlooks the planning and disbursement of finances on behalf
+of the project. The Astropy project accepts funds from institutions as well as
+individuals through NUMFOCUS\footnote{NumFOCUS is a 501(c)(3) nonprofit that
+supports and promotes world-class, innovative, open source scientific computing.},
+the fiscal sponsor of the project. Previously, no direct financial support
```
Comment (eteq) [inline src/ms.tex:300]:
```suggestion
supports and promotes world-class, innovative, open source scientific computing.}.
Previously, no direct financial support
```

I know this is what NF says themselves, but I think "fiscal sponsor" is often prone to misinterpretation as though they are giving us money as opposed to being more like the "bank".  (this goes with another comment below about NF).

File: src/ms.tex [hunk 12]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+was available for the project development, and NUMFOCUS covered the incurred
+operational costs. However, transitioning to long-term financial stability meant
+having an influx of funds in the form of institutional support \citep{2016arXiv161003159M},
+and hence the Astropy project applied for and successfully acquired several grants.
+The two major ones are briefly summarized below.
```
Comment (eteq) [inline src/ms.tex:305]: (to combine with the removal of "fiscal sponsor" above)
```suggestion
These grants are administered by NumFOCUS on behalf of the project, providing
a ``neutral'' space not tied to any particular astronomy institution.
The two major ones are briefly summarized below.
```

File: src/ms.tex [hunk 13]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+    improvements, (2) targeted work on areas of the \astropy core and
+    coordinated packages that need support, (3) enhancements to the Learn
+    Astropy ecosystem and educational materials (see \S \ref{sec:learn}),
+    and (4) support for the \astropy affiliated packages. These goals
+    support the sustenance of the project by encouraging further
+    development of the package ecosystem and community engagement.
+\end{itemize}
+
+In addition to the two grants described above, the project benefited from
+a Gemini Observatory contract under its Science User Support Department,
```
Comment (jdswinbank) [inline src/ms.tex:336]: Finance Ctte are going to run these two sentences past Gemini and Dunlap just to confirm they are happy with these mentions.

**Review by eteq — APPROVED** [review]
Overall looks great to me!  I have several inline wording/style suggestions, have a long and see if you agree with those.

File: src/ms.tex (hunk 6 above)
Comment (jdswinbank) [reply src/ms.tex:309]: Also, make the figures approximate (“~$900k”)

File: src/ms.tex [hunk 14]
```diff
@@ -287,9 +287,60 @@ \subsection{Funding} \label{sec:project-funding}
...
+    of equity, diversity, and inclusion efforts in the project (see \S
+    \ref{sec:project-ide}). Some of the money supported travel for meetings,
+    conferences, and workshops. This generous grant helped reduce reliance on
+    volunteer-driven maintenance of the openly-developed package and paved
+    the way to acquire funding from federal agencies: a big milestone for the
+    success of the project.
+
+    \item NASA: A successful proposal to the NASA ROSES-2020 call, section E.7
+    (Support for Open Source Tools, Frameworks, and Libraries), provided the
+    Astropy project \$634,590 to support its work. Funded work for a term of
```
Comment (jdswinbank) [inline src/ms.tex:325]: “~$600k”

File: src/ms.tex (hunk 10 above)
Comment (jdswinbank) [reply src/ms.tex:338]: Better: remove the figure.

**Comment by pllim** [comment]
Thank you, all! @aaryapatil , are you able to address the review comments and rebase? I think the build would pass after a rebase.

**Comment by aaryapatil** [comment]
@pllim Yes, will do so today! Sorry I was on leave last week, so didn't notice your previous comment.

**Comment by nmearl** [comment]
⚠️ 🚨 Just a heads up that the current plan is to have all open PRs merged by March 11th, in preparation for a final paper draft by March 18th. Please plan accordingly!

File: src/ms.tex (hunk 1 above)
Comment (aaryapatil) [reply src/ms.tex:301]: I took this from the Astropy v2.0 paper: https://iopscience.iop.org/article/10.3847/1538-3881/aabc4f/pdf -- 2.6. Support of Astropy "NumFOCUS has, to date, covered the direct costs incurred by the Astropy Project." Is that not correct? Should I use the word "most" or "some"?

**Comment by aaryapatil** [comment]
Comments have been addressed and build is passing. My editor messed up the spaces in references. Will fix soon.

File: src/ms.tex (hunk 13 above)
Comment (adrn) [reply src/ms.tex:338]: @jdswinbank Did you hear back?

Comment (jdswinbank) [reply src/ms.tex:338]: As I recall this was an action on @eteq, so perhaps he could commet.

Comment (adrn) [reply src/ms.tex:338]: I heard from @eteq in Slack that Gemini is happy as long as we add their standard acknowledgement -- I'll merge this and add the ack in a follow-up commit

“Supported by the international Gemini Observatory, a program of NSF’s NOIRLab, which is managed by the Association of Universities for Research in Astronomy (AURA) under a cooperative agreement with the National Science Foundation, on behalf of the Gemini partnership of Argentina, Brazil, Canada, Chile, the Republic of Korea, and the United States of America.”

**Comment by adrn** [comment]
Thanks @aaryapatil @jdswinbank and @eteq !

**Comment by aaryapatil** [comment]
Thanks @adrn 😄

**Pull request merged**